Formula/lean-cli.rb

```diff
@@ -0,0 +1,22 @@
+class LeanCli < Formula
+  desc "Command-line tool to develop and manage LeanCloud apps."
+  homepage "https://github.com/leancloud/lean-cli"
+  url "https://github.com/leancloud/lean-cli/archive/v0.2.0.tar.gz"
+  sha256 "980980c764486aa933255fd64d391560e72bba78af8427253e1b4515ab6b68eb"
+  head "https://github.com/leancloud/lean-cli.git"
+
+  depends_on "go" => :build
+
+  def install
+    build_from = build.head? ? "homebrew-head" : "homebrew"
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/leancloud/"
+    ln_s buildpath, buildpath/"src/github.com/leancloud/lean-cli"
+    system "go", "build", "-o", "output/lean", "-ldflags", "-X main.pkgType=#{build_from}", "github.com/leancloud/lean-cli/lean"
+    bin.install "output/lean"
+  end
+
+  test do
+    system bin/"lean", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Otherwise, let's at least assert:
```ruby
assert_match version.to_s, shell_output("#{bin}/lean --version")
```